arrangor/src/main/kotlin/no/nav/amt/tiltak/arrangor/ArrangorServiceImpl.kt

```diff
@@ -46,8 +49,30 @@ class ArrangorServiceImpl(
 		return arrangorRepository.getByOrganisasjonsnummer(virksomhetsnummer)?.toArrangor()
 	}
 
-	override fun getVirksomheterForAnsatt(ansattId: UUID): List<Arrangor> {
-		throw NotImplementedError("getVirksomheterForAnsatt in ArrangorService is not yet implemented")
+	override fun oppdaterArrangor(arrangorUpdate: ArrangorUpdate) {
+		val arrangor = getArrangorByVirksomhetsnummer(arrangorUpdate.organisasjonsnummer)
+
+		if (arrangor == null) {
+			// Sjekk om virksomheten er en overordnet enhet til noen arrangører og oppdater navn.
+			arrangorRepository.updateOverordnetEnhetNavn(arrangorUpdate.organisasjonsnummer, arrangorUpdate.navn)
+			return
```

Comment:
Litt snodig flyt at man skal oppdatere overordnet enhet hvis virksomheten ikke finnes isteden for å hente de som faktisk har den overordnede enheten og oppdatere de. Virker som man burde heller ta jobben med å innføre en overordnet_enhet_id som refererer til riktig enhet og så oppdatere bare den ene som alle de underordnede peker på